docs/manuals/auth/index.md

```diff
@@ -0,0 +1,77 @@
+!!! warning "Reminder: Authorization and Authentication is only available in the Enterprise version"
+
+# Pre-requisites
+
+An LDAP server must be set up by the user. Taipy provides no support for setting up the server.
+
+# Configuration
+
+You must configure how Taipy connects to the LDAP server using the Taipy `Config^` features. Here are the configurable properties you can use:
```

Comment:
Same here: that's true only if LDAP is used.

There should be a 'authentication_protocol' configuration option.

That can be one of 'none', 'ldap', or 'taipy'.

I suspect that 'auth_enabled' should be dropped (implemented by authentication_protocol == None/"none")

Reply:
would it be better to still have auth_enabled and authentication_protocol having a default value, or just return an error if no protocol is provided. I think it will be clearer that way

Reply:
I would recommend that the default value for 'authentication_protocol ' be None (the user does nothing).
That implies that auth_enabled is False.

That's because, if the users does want authorization, there is configuration to be done.

Reply:
okay sure, I agree. Will update the code in taipy-enterprise first, then push a commit on doc afterward.

Reply:
I am ok with this. We need to change that in core/enterprise

Reply:
ticket created here for taipy-enterprise: https://app.zenhub.com/workspaces/taipy-613ef328affde6000ff77fb3/issues/avaiga/taipy-enterprise/42